docs/conf.py

```diff
@@ -1,4 +1,4 @@
-extensions = ['sphinx.ext.autodoc', 'jaraco.packaging.sphinx', 'rst.linker']
+extensions = ['sphinx.ext.autodoc', 'jaraco.packaging.sphinx', 'rst.linker', 'sphinx_inline_tabs']
```

Comment:
This change is causing a nitpicky linter to fail the tests. I'd normally let you fix it, but I probably want to make some concession for other concerns (integration with the [upstream template](https://github.com/jaraco/skeleton)), so I can make the adjustments.

Reply:
I applied a fix in 265fb0eda.